File: CS-Examples/29_Print/SetMarginAndDuplex.cs
using System;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.IO;
using Spire.Doc;
using Spire.Doc.Documents;
using System.Drawing.Printing;

namespace SetMarginAndDuplex
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
    
            // Specify the input file path
            String input = @"..\..\..\..\..\..\Data\Sample.docx";

            // Create a new instance of Document
            Document doc = new Document();

            // Load the Word document from the specified input file
            doc.LoadFromFile(input);

            // Get the PrintDocument associated with the document
            PrintDocument printDoc = doc.PrintDocument;

            // Set the OriginAtMargins property to true to align the printable area with the margins
            printDoc.OriginAtMargins = true;

            // Set the Margins property of the DefaultPageSettings to zero to remove any margins
            printDoc.DefaultPageSettings.Margins = new System.Drawing.Printing.Margins(0, 0, 0, 0);

            // Set the Duplex property of PrinterSettings to Vertical for double-sided printing
            printDoc.PrinterSettings.Duplex = Duplex.Vertical;

            // Print the document
            printDoc.Print();

            // Dispose of the document object when finished using it
            doc.Dispose();
        }
    }
}

File: VB-Examples/29_Print/SetMarginAndDuplex.vb
Imports System.Text
Imports System.IO
Imports Spire.Doc
Imports Spire.Doc.Documents
Imports System.Drawing.Printing

Namespace SetMarginAndDuplex
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Specify the input file path
			Dim input As String = "..\..\..\..\..\..\Data\Sample.docx"

			' Create a new instance of Document
			Dim doc As New Document()

			' Load the Word document from the specified input file
			doc.LoadFromFile(input)

			' Get the PrintDocument associated with the document
			Dim printDoc As PrintDocument = doc.PrintDocument

			' Set the OriginAtMargins property to true to align the printable area with the margins
			printDoc.OriginAtMargins = True

			' Set the Margins property of the DefaultPageSettings to zero to remove any margins
			printDoc.DefaultPageSettings.Margins = New System.Drawing.Printing.Margins(0, 0, 0, 0)

			' Set the Duplex property of PrinterSettings to Vertical for double-sided printing
			printDoc.PrinterSettings.Duplex = Duplex.Vertical

			' Print the document
			printDoc.Print()

			' Dispose of the document object when finished using it
			doc.Dispose()
		End Sub
	End Class
End Namespace
